fineract-provider/src/main/java/org/apache/fineract/portfolio/search/api/SearchApiResource.java

```diff
@@ -85,8 +93,10 @@ public String retrieveAdHocSearchQueryTemplate(@Context final UriInfo uriInfo) {
     @GET
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String searchData(@Context final UriInfo uriInfo, @QueryParam("query") final String query,
-            @QueryParam("resource") final String resource ,@DefaultValue("false") @QueryParam("exactMatch")  Boolean exactMatch) {
+    @ApiOperation(value = "Search Resources", notes = "Example Requests:\n" + "\n" + "search?query=000000001\n" + "\n" + "\n" + "search?query=Petra&resource=clients,groups\n" + "\n" + "\n" + "search?query=Petra&resource=clients,groups&exactMatch=true")
+    @ApiResponses({@ApiResponse(code = 200, message = "", response = SearchApiResourceSwagger.GetSearchResponse.class)})
+    public String searchData(@Context final UriInfo uriInfo, @QueryParam("query")  @ApiParam(value = "query")final String query,
```

Comment:
Please add space between
```suggestion
    public String searchData(@Context final UriInfo uriInfo, @QueryParam("query")  @ApiParam(value = "query") final String query,
```